views/includes/page-templates/page-header.html

```diff
@@ -0,0 +1,21 @@
+{% if isUpdate %}
+  {% extends "update-layout.html" %}
+  {% set titleTemplate = "../update-page-title.html" %}
+{% else %}
+  {% extends "layout.html" %}
+  {% set titleTemplate = "../page-title.html" %}
+{% endif %}
+
+{% block pageTitle %}
+  {% include titleTemplate %}
+{% endblock %}
+
+{% block beforeContent %}
+  {% if isUpdate%}
+    {% include "../update_phase_banner.html" %}
+  {% else %}
+    {% include "../phase_banner.html" %}
+  {% endif %}
+  {% include "../no-back-link.html" %}
+  {% include "../sign-out-user-banner.html" %}
+{% endblock %}
```

Comment:
just a question - was it a requirement to add this beforeContent block or what does it do? Before your changes the `views/payment-failed.html` view didn't have this.

Reply:
sorry to be even more annoying now but does removing that code now get rid of the need for this new file? Would sonarqube complain about duplication anymore?

Reply:
i'd asked this same question but wasn't sure, might be worth a push to check and revert if it complains

Reply:
I'll check and see.

Reply:
Still complaining?? Ugh. Stupid Sonar. Just rename `titleTemplate` to something else, `pageTitle` maybe?